devicetypes/smartthings/testing/simulated-switch.src/simulated-switch.groovy

```diff
@@ -16,6 +16,8 @@ metadata {
     definition (name: "Simulated Switch", namespace: "smartthings/testing", author: "bob") {
 		capability "Switch"
         capability "Relay Switch"
+		capability "Sensor"
```

Comment:
**Yes!!!***

- Any Device that *sends* Events / Attribute values (i.e., reports a key State ... not counting something secondary like "Battery"), is a **Sensor**.

A **Switch** reports whether it is `on` or `off`. Perhaps better explained that *most* (or at least *many* Switches can be *physically* activated; thus, the Switch is "sensing" the activation and reporting it.

----------
By contrast, an "Actuator" is any Device that *receives* and acts upon Commands (again, except for secondary Commands like Configure, Poll, and Refresh).